packages/react/src/select/root/useSelectRoot.ts

```diff
@@ -168,7 +169,9 @@ export function useSelectRoot<T>(params: useSelectRoot.Parameters<T>): useSelect
     event: 'mousedown',
   });
 
-  const dismiss = useDismiss(floatingRootContext);
+  const dismiss = useDismiss(floatingRootContext, {
+    bubbles: false,
+  });
```

Comment:
Pressing `esc` in Select seems to still dismiss the outer Dialog in
https://deploy-preview-991--base-ui.netlify.app/experiments/popups-in-popups

Reply:
Yeah, I noticed. Also pressing outside closes all the popups at once.
It seems that this setting works only within a tree?

Reply:
`esc` should work without a tree, but outside press can't (though that's usually a good thing anyway in my view)